### PR TITLE
Bug 1483367 Remove Non-applicable Replication Content-2

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1483367

### CODE REVIEW

**Comment by bmcelvee** [comment]
Should line 96 change to `3306:3306 rhscl/mariadb-101-rhel7` or is that purely based on the example listed in: https://github.com/sclorg/mariadb-container/tree/master/10.1?

File: using_images/db_images/mariadb.adoc [hunk 1]
```diff
@@ -88,7 +88,7 @@ xref:../../architecture/core_concepts/pods_and_services.adoc#pods[pod] with
 MariaDB running in a container:
 
 ----
-$ oc new-app \
+$ docker run -d --name mariadb_database \
     -e MYSQL_USER=<username> \
     -e MYSQL_PASSWORD=<password> \
     -e MYSQL_DATABASE=<database_name> \
```
Comment (dongboyan77) [inline using_images/db_images/mariadb.adoc]: I think it is no need to modify command, just like below is ok:
`$ oc new-app \
    -e MYSQL_USER=<username> \
    -e MYSQL_PASSWORD=<password> \
    -e MYSQL_DATABASE=<database_name> \
    centos/mariadb-101-centos7`

**Comment by dongboyan77** [comment]
Others look good to me, thanks

**Comment by bmcelvee** [comment]
Thanks, @dongboyan77! 

@openshift/team-documentation PTAL 🍂

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review 🍁

**Comment by mburke5678** [comment]
@bmcelvee LGTM. You just removed a section, right? Do we need Dedicated and Online tags?

**Comment by bmcelvee** [comment]
@mburke5678 Yep! And yes, good catch! I'll add those tags.

**Comment by bmcelvee** [comment]
rev_history]
|xref:../using_images/db_images/mariadb.adoc#using-images-other-images-jenkins[using-images-db-images-mariadb]
|Removed non-applicable replication content.
%

**Pull request merged**